governance.md

```diff
@@ -59,260 +70,170 @@ The foundations of Project governance are:
-will be evaluated.
-
-If a Core Team member becomes inactive in the project for a period of one year,
-they will be considered for removal from the Core Team. Before removal,
-inactive Member will be approached by the BDFL to see if they plan on returning
-to active participation. If not they will be removed immediately upon a Core
-Team vote. If they plan on returning to active participation soon, they will be
-given a grace period of one year. If they don't return to active participation
-within that time period they will be removed by vote of the Core Team without
-further grace period. All former Core Team members can be considered for
-membership again at any time in the future, like any other Project Contributor.
-Retired Core Team members will be listed on the project website, acknowledging
-the period during which they were active in the Core Team.
-
-The Core Team reserves the right to eject current Members, other than the BDFL,
-if they are deemed to be actively harmful to the project’s well-being, and
-attempts at communication and conflict resolution have failed.
+To manage the project, there are different Teams that each have responsibility for
+specific aspects of the project. Collectively, the members of all Teams are referred to
+as _Stewards_ of the project.
+
+## Steering Committee
+
+The role of the _pandas_ Steering Committee is to coordinate the activities of the
+different Teams and to ensure that different policies and procedures are carried out in
+a consistent manner. The Steering Committee has no routine decision-making authority,
+except as detailed herein, although in exceptional circumstances it may be called upon
+from time to time to make decisions that are in the best interest of The Project as a
+whole. The Steering Committee will itself decide when a circumstance is exceptional.
+
+The Steering Committee may create a working group to consider changes to the governance
+model described in this document, including the creation of new Teams that support the
+goals of The Project. Changes to the governance model will require a voting process and
+approval equivalent to the process described in PDEP-1, with the exception that all
+Stewards are eligible to vote.  
+
+The Steering Committee may appoint temporary working groups to work on issues, such as
+governance, that fall outside the scope of responsibilities for the existing Teams.
+
+The ideal composition of the Steering Committee consists of 5 people.  The Steering
+Committee will be chosen via an approval voting process from a slate of nominees,
+meaning that each person who is eligible to vote may vote for more than one candidate
+from the slate.  
+
+### Initial Election
+
+The initial slate for the Steering Committee election will consist of 5 or more
+candidates who volunteer to be on the slate of candidates for the initial Steering
+Committee.  For the initial election, any candidate receiving more than 50% of the total
```

Comment:
I don't understand this sentence: 

> For the initial election, any candidate receiving more than 50% of the total
number of members of the previous _pandas_ core team that register at least one vote
will then become a member of the initial Steering Committee.

Reply:
The idea here is that the current core team does the initial vote.  Let's say only 15 people from the current core team vote.  Then any candidate receiving 8 or more votes is on the initial Steering Committee.  

Open to suggestions for clarifying this.

Reply:
> that register at least one vote

This read to me as 1 vote would be sufficient to be voted in. Your clarification makes more sense, thanks.

Maybe:

> For the initial election, any candidate receiving a vote from more than 50% of the total
number of members of the previous pandas core team who voted in the election
will then become a member of the initial Steering Committee.

Reply:
@Dr-Irv I forgot again why we wanted to do this initial election differently. What was the reason to not follow the procedure as described for subsequent years? i.e. we have a set of candidates, do approval voting, and top-5 are elected?

Reply:
The concern here is that we might have a lot of people who want to be on the Steering Committee who are currently on the core team, so we wanted to not exclude anyone.  Conversely, if we didn't have enough people who wanted to be on the Steering Committee, it would be easier to get approval of half of the people.

I'm not opposed to using the future "top 5" process, but we should also determine the threshold for how many of the current core team members need to vote.  Or we don't worry about that - if only 10 of the core team members vote, we take the top 5 from that.

Reply:
After discussion, we'll change this so that the initial election is a top 5 vote getters among the current pandas core team that votes.

Reply:
fixed in next revision